views/modal/wp-editor-link.php

```diff
@@ -13,31 +13,31 @@
 <div id="wp-link-wrap" class="wp-core-ui" style="display: none" role="dialog" aria-labelledby="link-modal-title">
 	<form id="wp-link" tabindex="-1">
 		<?php wp_nonce_field( 'internal-linking', '_ajax_linking_nonce', false ); ?>
-		<h1 id="link-modal-title">Insert/edit link</h1>
+		<h1 id="link-modal-title"><?php echo esc_html__( 'Insert/edit link' ); ?></h1>
```

Comment:
Need to use like the following:
`?php esc_html_e( 'Insert/edit link', 'tutor' ); ?>`